### DISCUSSION TITLE
Unable to log into https://demo.netbox.dev

### DISCUSSION THREAD

**Discussion by WhaleJ84** [netbox-community/netbox#8004] [General]
Hey all,

Both today and yesterday (07 - 08 Dec 2021) I have tried to log into the [demo instance](https://demo.netbox.dev) to look around the new features in v3.1.0.
The banner on the page states the username is **admin** and the password is **admin**.
Typing in those credentials exactly the same (all lower-case), it shows an error message of:

*'Please enter a correct username and password. Note that both fields may be case-sensitive.'*

Has the credentials changed without the banner being updated? Is someone logging in after the daily refreshes and changing the password? Is this issue just me?

I'm curious to see if anyone else is facing this issue, as I don't see it reported elsewhere yet.

**Comment by jeremystretch** [comment]
There's an issue loading the demo data (which includes the admin user) from its v3.0 format. I need to adapt it for v3.1, then it should should work as expected.

**Reply by jeremystretch** [reply]
All fixed now!

**Reply by WhaleJ84** [reply]
Yeah, seems to work fine now. Thanks!
